src/pages/result/index.tsx

```diff
@@ -1,53 +1,99 @@
 import styled from '@emotion/styled'
-import type { NextPage } from 'next'
+import { useAtomValue } from 'jotai'
+import type { GetServerSideProps, NextPage } from 'next'
+import { useRouter } from 'next/router'
 import { useState } from 'react'
 import type {
   SearchOptionsState,
   OnChangeSearchOptions
 } from '@components/result/SearchOptions/types'
 import { useGetCategoriesQuery, useGetInfinitePostsQuery } from '@apis'
+import { searchKeywordAtom } from '@atoms'
 import {
   SearchOptions,
   ResultHeader,
   CategorySlideFilter,
   ProductList
 } from '@components'
+import type { SortOptionCodes, TradeTypeCodes } from '@types'
+import { toQueryString, removeUndefinedAndNull } from '@utils'
 
 const DEFAULT_POST_PAGE_NUMBER = 8
 
-const ResultPage: NextPage = () => {
+type Props = {
```

Comment:
```suggestion
type ResultPageProps = {
```
어떤 props인지 명시해주면 좋을 것 같습니다!

Reply:
[반영 완료](https://github.com/price-offer/offer-fe/pull/241/commits/4dc6b204eaad5c8f3b150fc45139a90e9d72c449#diff-33ad00554e4d8335fcb7f35a52e38907ad21791c810c9615fe473c46ff271abcR23) !